docs/README.md

```
@@ -0,0 +1,82 @@
+# OSLC4Net Documentation
+
+This directory contains the source files for the OSLC4Net documentation site, which is published to [oslc4net.github.io](https://oslc4net.github.io).
+
+## Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)